### PR TITLE
Angular constants via WebSocket

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
- new format for constants on the server
- adaptions for the old client

### CODE REVIEW

File: openslides/core/apps.py [hunk 1]
```diff
@@ -110,17 +110,13 @@ def get_angular_constants(self):
...
-            'name': 'OpenSlidesConfigVariables',
-            'value': config_groups}
+        constants['OpenSlidesConfigVariables'] = config_groups
 
         # Send the privacy policy to the client. A user should view them, even he is
         # not logged in (so does not have the config values yet).
-        privacy_policy = {
-            'name': 'PrivacyPolicy',
-            'value': config['general_event_privacy_policy']}
+        constants['PrivacyPolicy'] = config['general_event_privacy_policy']
```
Comment (ostcar) [inline openslides/core/apps.py]: Remove this config var and create a privacy policy at another place. (See the comment in consumers.py).

File: openslides/utils/consumers.py [hunk 2]
```diff
@@ -136,6 +137,19 @@ class SiteConsumer(ProtocollAsyncJsonWebsocketConsumer):
...
+            angular_constants: Dict[str, Any] = {}
+            for app in apps.get_app_configs():
+                try:
+                    # Each app can deliver values to angular when implementing this method.
+                    # It should return a list with dicts containing the 'name' and 'value'.
+                    get_angular_constants = app.get_angular_constants
+                except AttributeError:
+                    # The app doesn't have this method. Continue to next app.
+                    continue
+                constants = await database_sync_to_async(get_angular_constants)()
```
Comment (ostcar) [inline openslides/utils/consumers.py]: I want the SiteConsumer to be fully async without any switch to sync (which starts new threads). Specially database requests in another thread are problematic, see: https://github.com/django/channels/issues/1110  

the angular_constants should be real constants. Therefore I would suggest, that you load all constants into a global dict on startup. Then you only have to return this dict here.

To make it even clearer, that the constants are constants, I would say, we remove the config vars from the constants. They are problematic. For example when you render the index.html the constants are hard coded into the rendered file. When you change the config vars afterwards, the change has no effect, which can be confusing.

Therefore "variable" constants should be in the settings.py to make in clearer, that they can not be changed at runtime.

File: openslides/core/apps.py (hunk 1 above)
Comment (tsiegleauq) [inline openslides/core/apps.py]: Fails for me using a clean setup.
`The config variable general_event_privacy_policy was not found`

Is seems that `get_config_variables()` was never called

**Comment by FinnStutzenstein** [comment]
@ostcar This is ok, the `privacy_policy` was a hack for the old client. I moved this information to the login GET request, where also other relevant information for the login page is delivered.
 If you approve, this can be merged.

**Comment by FinnStutzenstein** [comment]
@ostcar Can you check the python code and the error in the tests?

**Pull request merged**